### DISCUSSION TITLE
Add Option to disable "find" dialog (Ctrl+F)

### DISCUSSION THREAD

**Discussion by mattelacchiato** [outline/outline#6465] [Ideas]
When I press Ctrl+F to find in the current page, outline overwrites the browser shortcut and displays its own find-popup. This is a little bit annoying, since e.g. other shortcuts like F3 (jump to next occurence) does not work here.

For me, the outline popup does not give any benefit, so I would like to disable it.

(Current workaround: When pressing Ctrl+F twice, the browser find dialog opens)

**Comment by tommoor** [comment]
Probably not going to add an option for this as you can get to the native find with CMD+F twice as you mentioned. The inbuilt find has the ability to replace content in the document and regex match, neither of which are available in browser.

I just deployed a change to fix the native shortcuts for back and forward, I wasn't aware of them but they should work now.
